ReferenceBackendAPI.md

```diff
@@ -0,0 +1,193 @@
+
+Request parameters:
+
+```json
+{
+  "user_identity": "Bob",
+  "stream_name": "demo"
+}
+```
+
+Example response:
+
+```json
+{
+  "token": "xxxx.yyyyy.zzzzz"
+}
+```
+
+### /join-stream-as-speaker
+
+This endpoint adds the user to the Twilio conversation and adds some user state to the sync objects. The endpoint returns a [Twilio access token](https://www.twilio.com/docs/iam/access-tokens). The token includes grants that allow the client to connect to the video room, sync objects, and conversation. The endpoint should be called when a user initially joins the stream as a speaker or when a user transitions from viewer to speaker.
+
+Request parameters:
+
+```json
+{
+  "user_identity": "Bob",
+  "stream_name": "demo"
+}
+```
+
+Example response:
+
+```json
+{
+  "token": "xxxx.yyyyy.zzzzz"
+}
+```
+
+### /join-stream-as-viewer
+
+This endpoint adds the user to the Twilio conversation and adds some user state to some sync objects. The endpoint returns a [Twilio access token](https://www.twilio.com/docs/iam/access-tokens). The token includes grants that allow the client to connect to the live stream, sync objects, and conversation. 
+The endpoint should be called when a user initially joins the stream as a viewer or when a user transitions from speaker to viewer.
+
+Request parameters:
+
+```json
+{
+  "user_identity": "Bob",
+  "stream_name": "demo"
+}
+```
+
+Example response:
+
+```json
+{
+  "token": "xxxx.yyyyy.zzzzz"
+}
+```
+
+### /viewer-connected-to-player
+
+This endpoint updates some user state in the sync objects to reflect that the user is connected to the stream. The client should call this endpoint after a viewer successfully connects to the stream using the Twilio Live Player SDK.
+
+Request parameters:
+
+```json
+{
+  "user_identity": "Bob",
+  "stream_name": "demo"
+}
+```
+
+Example response:
+
+```json
+{
+  "success": true
+}
+```
+
+### /raise-hand
+
+This endpoint updates sync objects to reflect that a user has either raised or lowered their hand. A client should call this endpoint when a viewer wants to signal to the host that they want to speak. The endpoint can also be called when a user changes their mind and wants to lower their hand.
+
+Request parameters:
+
+```json
+{
+  "user_identity": "Bob",
+  "stream_name": "demo",
+  "hand_raised": true
+}
+```
+
+Example response:
+
+```json
+{
+  "sent": true
+}
+```
+
+### /remove-speaker
+
+This endpoint removes a user from the video room. The endpoint should be called when a host wants to remove a speaker that is misbehaving.
```

Comment:
```suggestion
This endpoint removes a user from the video room. The endpoint should be called when a host wants to remove a speaker that causing shenanigans.
```
Just for fun 🙂 if you want. Also consistent with the `X-Shenanigans: none` header seen on the serverless responses.

Reply:
Oh yeah that is more Twilio.